### DISCUSSION TITLE
Last 📢 news for the next 1.23 major update is out! 🍾

### DISCUSSION THREAD

**Discussion by Vdauphin** [Vdauphin/HeartsAndMinds#1451] [Announcement]
# 👋 Hey! 
Here is some 📢**news** about our favorite mission: https://forums.bohemia.net/forums/topic/165948-mp-btc-hearts-and-minds/?page=75&tab=comments#comment-3469051

### And remember: :heart: and :brain:!

**Comment by sancron** [comment]
And where do we download the update? Only 1.22.8 is currently available on the release page.

And thank you for the work you put into this.

**Reply by Vdauphin** [reply]
Hello, 
The release is planned here: https://github.com/Vdauphin/HeartsAndMinds/milestone/73
Cheers

**Reply by sancron** [reply]
Thank you ❤️
